src/v2/Artsy/Analytics/Schema/index.ts

```diff
@@ -18,6 +18,14 @@ import { AuthenticationInteraction, Interaction } from "./Interaction"
 import { Label } from "./Label"
 import { Failure, Success } from "./Result"
 import { Type } from "./Type"
+import {
+  ClickedArtistGroup,
```

Comment:
All the cohesion events are available as `Event` so you can add them all to the schema at once: https://github.com/artsy/cohesion/blob/master/src/Schema/Events/index.ts#L31